### PR TITLE
[Xcodeproj] Generating clibs targets 2

### PR DESCRIPTION
**Pull request by goloveychuk**
Replaces #205 
Generated for project with 20 deps - everything works. 
Please review.

### CODE REVIEW

File: Sources/Xcodeproj/pbxproj().swift [hunk 1]
```diff
@@ -15,9 +15,10 @@
 import PackageType
 import Utility
 
-public func pbxproj(srcroot srcroot: String, projectRoot: String, modules: [SwiftModule], externalModules: [SwiftModule], products _: [Product], printer print: (String) -> Void) {
+public func pbxproj(srcroot srcroot: String, projectRoot: String, modules: [XcodeModuleProtocol], externalModules: [XcodeModuleProtocol], products _: [Product], printer print: (String) -> Void) {
 
-    let rootModulesSet = Set(modules).subtract(Set(externalModules))
+    // let rootModulesSet = Set(modules).subtract(Set(externalModules))
```
Comment (goloveychuk) [inline Sources/Xcodeproj/pbxproj().swift]: The only problem is this 2 lines. As I understood, I can't do this for protocols?

**Comment by paulofaria** [comment]
@mxcl It works beautifully here with https://github.com/Zewo/Zewo, which has 34 dependencies and 3 of them are C libs.

**Comment by mxcl** [comment]
@swift-ci Please test

**Comment by goloveychuk** [comment]
@mxcl I fixed test case, please test again.

**Comment by ddunbar** [comment]
@swift-ci please test

**Comment by mxcl** [comment]
@swift-ci Please test

**Comment by paulofaria** [comment]
```
/Users/buildnode/jenkins/workspace/swift-package-manager-PR-osx/swift/stdlib/public/SDK/WatchKit/WatchKit.swift:38:5: error: value of type 'WKInterfaceController' has no member 'present'
```

`WatchKit`?

**Comment by goloveychuk** [comment]
Ok, build succeed. The problem, as I understand, was with main swift codebase.

**Pull request merged**